test/e2e/common/node/image_credential_provider.go

```diff
@@ -1,69 +0,0 @@
-/*
-Copyright 2022 The Kubernetes Authors.
```

Comment:
@ndixita is there any other tests covering creds provider e2e? 

/hold

Reply:
there are no other tests covering the credential provider.